uaclient/tests/test_util.py

```diff
@@ -780,6 +780,7 @@ class TestIsConfigValueTrue:
         [
             ({}, False),
             ({}, False),
+            ({None}, False),
```

Comment:
Do we need the curly braces here?
```suggestion
            (None, False),
```